src/schema/objects/top_level_files.yaml

```
@@ -21,9 +21,18 @@ LICENSE:
 README:
   name: README
   description: |
-    In addition a free form text file (`README`) describing the dataset in more details SHOULD be
-    provided.
-    The `README` file MUST be either in ASCII or UTF-8 encoding.
+    A REQUIRED text file, `README`, SHOULD describe the dataset in more detail.
+    The `README` file MUST be either in ASCII or UTF-8 encoding and MAY have one of the extensions:
+    `.md` ([Markdown](https://www.markdownguide.org/)),
+    `.rst` ([reStructuredText](https://docutils.sourceforge.io/rst.html)),
+    or `.txt`.
```

it is somewhat implied but may be worth explicit something like:
```suggestion
    or `.txt`.
    Only a single README file (with or without extension) should be present.
```
?

---

agreed, though I would phrase it:

> a BIDS dataset MUST NOT contain more than one README file (with or without extension) at its root directory.

---

Fixed.